Project-Descriptions-and-Plans/FORCE-RISCV/FORCE-RISCV-Project-Launch-revision2023.md

```diff
@@ -0,0 +1,285 @@
+  - *TWG grants PC gate with feedback, or rejects PC gate with feedback*
+  - *If PC granted, additional work is carried out to prepare the **PL proposal***. 
+- *The **PL proposal** contains updates to the PC fields and adds additional fields. The PL proposal explains the "what" of the project.* 
+  - *For software porting projects, the PL should contain the feature list*
+  - *For IP core or other complex projects, the PL should contain a high level feature list (the user manual with feature specification is developed for the Plan Approved gate).*
+
+
+<hr/>
+
+***Part 1, PC fields:***
+*The PC proposal introduces the project and explains the market drivers and the "why"* 
+
+# "FORCE-RISCV ISG"
+# Project Concept Proposal
+## Date of proposal 
+- 2023-03-6
+
+## Author(s) 
+- Binguang.Zhao/Ethan
+
+## High Level Summary of project, project components, and deliverables
+FORCE-RISCV is a powerful verification tool for RISC-V based cpu design.
+It has been developed in Futurewei Technologies and delivered to OpenHW Group on June 12 2020. 
+With the 1st stage work from Futurewei Technologies, FORCE-RISCV has made built a good isg framework.
+But lots of features need to be updated or upgraded with the booming development of riscv architecture. For example, new extensions need to be supported or existing extensions should be updated.
+
+Albaba T-HEAD XuaTie team (XuanTie team for short) takes charge of the stage 2 development work of FORCE-RISCV with the open source community.
+
+FORCE-RISCV currently supports 64-bit/32bit version of RISC-V with RV64I, M, A, Zicsr, F, D, C extensions, Machine, Supervisor, User modes, dynamic ISS interaction, dynamic Virtual Memory management, loops, configurable state transition, powerful and extensible Python scriptable test template writing framework etc. Vector extension 1.0 support, full paging exception control, basic MP/MT support and advanced resource dependency are also supported right now.
+
+Many of the current OpenHW Group member's verification need can be satisfied with the powerful features provided by FORCE-RISCV.
+
+## Summary of market or input requirements
+### Known market/project requirements at PC gate
+- No additional resource requirements yet identified
+
+### Potential future enhancements
+New features requested by OpenHW Group memebers and broader open source community.
+
+## Who would make use of OpenHW output
+CORE-V Vision APU project will be able to start using FORCE-RISCV with vector extension support.
+OpenHW Group members working on 64-bit and/or 32-bit RISC-V processor designs will directly benefit from the FORCE-RISCV with both 64-bit and 32-bit support in the long term.
+
+## Initial Estimate of Timeline
+TODO:
+
+
+## Explanation of why OpenHW should do this project
+The following strengths of FORCE-RISCV are in comparison to other open source ISGs where FORCE-RISCV does it better: 
```

Comment:
There are some non-printable characters in this list.  Recommend you delete these characters and use "-".

Reply:
I have reformat the file. Please check again.